### PR TITLE
overwritten checkout theme templates with confira styling

### PR DESCRIPTION
**Pull request by jordythevulder**

### CODE REVIEW

File: resources/views/cart/overview.blade.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@section('robots', 'NOINDEX,NOFOLLOW')
+
+@section('content')
+    <cart v-cloak>
+        <div v-if="hasItems" class="container" slot-scope="{ cart, hasItems, changeQty, remove }">
+            <x-rapidez-ct::title class="mb-5 text-ct-primary">
+                @lang('Cart')
+            </x-rapidez-ct::title>
+            <x-rapidez-ct::layout class="[&>div:first-child>div>table]:my-0">
```
Comment (jordythevulder) [inline resources/views/cart/overview.blade.php:13]: i've added this into the checkout theme as a PR, this is just for the timebeing. i will monitor the pullrequest, and if it gets merged i'll personally delete this.

File: resources/views/components/layout/checkout.blade.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@slots(['header', 'sidebar'])
+<div class="fixed -top-px h-1.5 left-0 right-0 w-full bg-primary z-20"></div>
```
Comment (Roene-JustBetter) [inline resources/views/components/layout/checkout.blade.php:2]: `inset-x-0`

File: resources/views/components/layout/checkout.blade.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+<div {{ $attributes->class('text-ct-primary flex flex-wrap gap-x-20 text-sm max-md:flex-col') }}>
+    <div class="flex flex-wrap lg:flex-row flex-col flex-1 max-lg:pt-6">
```
Comment (Roene-JustBetter) [inline resources/views/components/layout/checkout.blade.php:4]: default flex behaviour is `flex-row` so you can change this to `max-lg:flex-col`

File: resources/views/components/layout/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
+@slots(['header', 'sidebar'])
+
+<div {{ $attributes->class('text-ct-primary flex flex-wrap gap-x-28 text-sm max-md:flex-col') }}>
+    <div class="flex flex-wrap lg:flex-row flex-col flex-1">
```
Comment (Roene-JustBetter) [inline resources/views/components/layout/index.blade.php:4]: default flex behaviour is `flex-row `so you can change this to `max-lg:flex-col`

**Review by Roene-JustBetter — CHANGES_REQUESTED** [review]

**Review by Roene-JustBetter — APPROVED** [review]

File: resources/views/components/card/inactive.blade.php [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<div {{ $attributes->class('') }}>
```
Comment (royduin) [inline resources/views/components/card/inactive.blade.php:1]:
```suggestion
<div {{ $attributes }}>
```

File: resources/views/components/card/index.blade.php [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div {{ $attributes->class('') }}>
```
Comment (royduin) [inline resources/views/components/card/index.blade.php:1]:
```suggestion
<div {{ $attributes }}>
```

**Review by royduin — CHANGES_REQUESTED** [review]

**Pull request merged**